02/app.js

```diff
@@ -0,0 +1,57 @@
+const registerForm = document.querySelector("form");
+registerForm.noValidate = true;
+
+const checkForm = function (e) {
+  e.preventDefault();
+
+  const emailInput = e.target.elements.login;
+  const passwordInput = e.target.elements["pass1"];
+  const passwordConfirmInput = e.target.elements["pass2"];
+  const acceptCheckbox = e.target.elements["accept"];
+
+  const errors = [];
```

Comment:
Resetujesz błędy, ale nie resetujesz koloru dla label ;)
Jeśli za pierwszym razem wpiszę coś błędnie, a potem to poprawie to i tak ciągle jest na czerwono ;)